core/common/src/main/java/alluxio/conf/PropertyKey.java

```diff
@@ -4823,13 +4833,15 @@ public String toString() {
               + "authentication is enabled. Server trusts whoever the client claims to be.")
           .setConsistencyCheckLevel(ConsistencyCheckLevel.ENFORCE)
           .setScope(Scope.ALL)
+          .setIsDynamic(false)
```

Comment:
there shall be some other configurations, like master RPC port, `MASTER_HOSTNAME`, `MASTER_METASTORE`, journal type (UFS, Raft), `HOME`, and many others to be non-dynamic. Now I actually feel it is safer to disable updating a property, unless we are sure it can be set dynamically. (e.g., these timeout). We can discuss it offline

Reply:
OK